scripts/make-packages.sh

```diff
@@ -184,6 +184,12 @@ if [ "$OS" == "linux" ]; then
      FILES="${FILES} ${ARTIFACTS}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}=/usr/${LIBDIR}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}"
   fi
 
+  # XDP means Ubuntu 24.04 which installs libssl3t64 instead of libssl3
```

Comment:
Is there a case where we build a package for ubuntu 24.04 without depending on XDP?